tests/data_files/Makefile

```diff
@@ -13,8 +13,10 @@
 ## Tools
 OPENSSL ?= openssl
 FAKETIME ?= faketime
-MBEDTLS_CERT_WRITE ?= $(PWD)/../../programs/x509/cert_write
-MBEDTLS_CERT_REQ ?= $(PWD)/../../programs/x509/cert_req
+
+TOP_DIR = $(dir $(firstword $(MAKEFILE_LIST)))/../..
```

Comment:
Oh, right. `$(PWD)` isn't right because it comes from the shell, not from make, so it's wrong when `-C` is used. Using `$(MAKEFILE_LIST)` is actually wrong because it's the location of the makefile, not where we want to act, so it would break `make -f ../../../another_branch/tests/data_files/Makefile`. I'll fix this.